hack/update-kcp-codegen.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+unset GO111MODULE
+
+#kube::golang::setup_env
+
+# go install github.com/kcp-dev/code-generator
```

Comment:
This part is still TODO, we do need to install code-generator (or at least fail early and point at the repo)